.github/workflows/docs-and-package.yaml

```diff
@@ -25,35 +25,39 @@ jobs:
           git config user.name "github-actions[bot]"
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
-      - name: Guard - Commit message should have prefix
-        id: guard_commit_message
+      - name: Guard - BREAKING CHANGE should not be in body
+        id: guard_breaking_change
         run: |
           LAST_COMMIT_MESSAGE="${{ github.event.head_commit.message }}"
           echo "Commit message: \"$LAST_COMMIT_MESSAGE\""
-          if [[ "$LAST_COMMIT_MESSAGE" != feat:\ * && "$LAST_COMMIT_MESSAGE" != fix:\ * && "$LAST_COMMIT_MESSAGE" != chore:\ * && "$LAST_COMMIT_MESSAGE" != docs:\ * ]]; then
-            echo "ERROR: Prefix 'feat: ', 'fix: ', 'docs: ' or 'chore: ' is missing from the commit message."
-            exit 1
+          if [[ "$LAST_COMMIT_MESSAGE" == *"BREAKING CHANGE"* ]]; then
+              echo "ERROR: Commit message should not include 'BREAKING CHANGE'."
+              exit 1
           fi
         shell: bash
 
-      - name: Guard - BREAKING CHANGE should not be in body
-        id: guard_breaking_change
+      - name: Guard - Decide if Package & Docs should run
+        id: guard_job
         run: |
           LAST_COMMIT_MESSAGE="${{ github.event.head_commit.message }}"
           echo "Commit message: \"$LAST_COMMIT_MESSAGE\""
-          if [[ "$LAST_COMMIT_MESSAGE" == *"BREAKING CHANGE"* ]]; then
-            echo "ERROR: Commit message should not include 'BREAKING CHANGE'."
-            exit 1
+          if [[ "$LAST_COMMIT_MESSAGE" != feat:\ * && "$LAST_COMMIT_MESSAGE" != fix:\ * ]]; then
+              echo "Commit message does not include 'feat: ' not 'fix: '. Package & Docs pipeline will not run."
```

Comment:
Minor unimportant detail: Should `not` be `nor` in `does not include 'feat: ' not 'fix: '.`